test/run_test.py

```diff
@@ -1,345 +1,345 @@
-#!/usr/bin/env python
```

Comment:
it looks like you changed something in this file that you shouldn't have.
I looked, and it's whitespace changes.

Can you revert changes to this file entirely.

Reply:
Whoops, yeah. My editor is usually good about preserving line endings and such, but I upgraded versions yesterday, so perhaps some settings got changed.

I'll revert in a moment.